client/components/deposits-overview/deposit-notices.tsx

```diff
@@ -197,7 +197,7 @@ export const NoFundsAvailableForDepositNotice: React.FC = () => (
 	<InlineNotice status="warning" icon isDismissible={ false }>
 		{ interpolateComponents( {
 			mixedString: __(
-				'You have no funds available to deposit. {{whyLink}}Why?{{/whyLink}}',
+				'You have no funds available. {{whyLink}}Why?{{/whyLink}}',
```

Comment:
Need to review this in context to confirm the new wording works, and/or get design or product feedback.

Reply:
I confirm that the removal of those words `to deposit` still makes sense since `<NoFundsAvailableForDepositNotice />` is currently only displayed on the overview page within the Deposit section ([ref](https://github.com/Automattic/woocommerce-payments/blob/d7861f8df73a7b5666d07e653c6c588199db8309/client/components/deposits-overview/index.tsx#L164)).

![Screenshot 2024-10-22 at 01 53 26](https://github.com/user-attachments/assets/0b1c17df-508b-4e63-a703-4d5b49a2eaed)